src/dispatch/feedback/service.py

```diff
@@ -14,6 +15,19 @@ def get_all(*, db_session):
     return db_session.query(Feedback)
 
 
+def get_all_last_x_hours(
+    *, db_session, hours: int = 24, skip=0, limit=100
+) -> List[Optional[Feedback]]:
+    """Returns all feedback provided in the last x hours."""
+    return (
+        db_session.query(Feedback)
+        .filter(Feedback.created_at >= datetime.utcnow() - timedelta(hours=hours))
+        .offset(skip)
+        .limit(limit)
```

Comment:
Why do we limit this? You don't seem to use pagination anywhere. Same with skip/offset.

Reply:
Correct, not needed, will remove.